docs/source/api/to_extensions.rst

```diff
@@ -103,6 +103,9 @@ Response Structure
 
 ``POST``
 ========
+.. versionchanged:: ATCv4
```

Comment:
This only applies to the 1.5 handler, right? I think here and below it should name the API version in which it changed, because if you request an older version you will still get the old behavior, even in ATCv4.